spryker/dockersuite_staging.yml

```diff
@@ -256,3 +247,13 @@ sections:
             command: "vendor/bin/console frontend:zed:build"
             groups:
                 - zed
+
+    jenkins-up:
+        jenkins-generate:
```

Comment:
The generating of jobs was commented in the previous version.

Reply:
In previous versions we call jenkins:generate from setup_suite.sh script. I'v remove it from there and added here.